Automattic/wp-calypso
# Fetch Podcast Title From Gutenboarding

#### Changes proposed in this Pull Request

* If we have a podcast ID, request the podcast title from an API endpoint that parses the podcast RSS.

#### Testing instructions

* Checkout this branch
* `ENABLE_FEATURES=anchor-fm-dev` if necessary
* Go to http://calypso.localhost:3000/new?anchor_podcast=3e51070 as both a new user and a signed-in user and check that the podcast title is pre-filled as the site title:
<img width="1412" alt="Screen Shot 2020-12-16 at 4 04 59 PM" src="https://user-images.githubusercontent.com/1689238/102406373-771f3b80-3fb8-11eb-835d-7a6c5f844782.png">

* Also verify that the username (if a new one is created) and the site slug/domain (e.g. allisonspodcast.wordpress.com) reflect the podcast title.
